File: java_backup/my java/bishwarup/HCF.java
class HCF
{
public static void main(int n)
{
int i,HCF=0,a=0,b=0;
for(i=1;i<=n/2;i++)
{
if(a%i==0&&b%i==0)
HCF=i;
}

System.out.print("H.C.F" + HCF);
System.out.print("L.M.C" + a*b + HCF);
}
}

File: 4rth_Sem_c++_backup/SHIT/hcf.cpp
#include"iostream.h"
#include<conio.h>
int main()
{
   //clrscr();
   int a,b,c;
   cout<<"enter hcf for two no"<<endl;
   cin>>a;
   cin>>b;
   while(a%b!=0)
    {
     c=a%b;
     a=b;
     b=c;
    }
   cout<<b;
   getch();
   return 0;
}    
